src/Directory.Packages.props

```diff
@@ -0,0 +1,28 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <ArcGISMapsSDKVersion>200.2.0</ArcGISMapsSDKVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Esri.ArcGISRuntime" Version="$(ArcGISMapsSDKVersion)" />
+	<PackageVersion Include="Esri.ArcGISRuntime.Toolkit.WPF" Version="$(ArcGISMapsSDKVersion)" />
+    <PackageVersion Include="Esri.ArcGISRuntime.WPF" Version="$(ArcGISMapsSDKVersion)" />
+    <PackageVersion Include="Esri.ArcGISRuntime.Maui" Version="$(ArcGISMapsSDKVersion)" />
+    <PackageVersion Include="Esri.ArcGISRuntime.Toolkit.Maui" Version="$(ArcGISMapsSDKVersion)" />
+	<PackageVersion Include="Esri.ArcGISRuntime.WinUI" Version="$(ArcGISMapsSDKVersion)" />
+    <PackageVersion Include="Esri.ArcGISRuntime.Hydrography" Version="$(ArcGISMapsSDKVersion)" />
+	<PackageVersion Include="Esri.ArcGISRuntime.LocalServices" Version="100.15.0" />
+	<PackageVersion Include="Markdig" Version="0.30.4" />
+	<PackageVersion Include="System.Text.Json" Version="7.0.1" />
+    <PackageVersion Include="System.Speech" Version="7.0.0" />
+	<PackageVersion Include="System.Drawing.Common" Version="7.0.0" />
+    <PackageVersion Include="System.Security.Cryptography.ProtectedData" Version="7.0.0" />
+    <PackageVersion Include="CommunityToolkit.WinUI.UI.Controls" Version="7.1.2" />
+    <PackageVersion Include="CommunityToolkit.WinUI.UI.Controls.Markdown" Version="7.1.2" />
+	<PackageVersion Include="CommunityToolkit.Maui" Version="5.0.0" />
+    <PackageVersion Include="CommunityToolkit.Mvvm" Version="8.1.0" />
+	<PackageVersion Include="WinUIEx" Version="1.8.0" />
+    <PackageVersion Include="System.Security.Cryptography.ProtectedData" Version="7.0.0" />
+	<PackageVersion Include="Xamarin.AndroidX.AppCompat" Version="1.5.1.1" />
```

Comment:
Do we need to centrally manage packages that are only used in a single project? (goes for several here)